src/lib/stitching/mergeSchemas.ts

```diff
@@ -95,6 +96,28 @@ export const incrementalMergeSchemas = (localSchema, version: 1 | 2) => {
     useStitchingEnvironment(kawsStitchingEnvironmentV2(localSchema, kawsSchema))
   }
 
+  if (version === 2) {
+    useStitchingEnvironment({
+      extensionSchema: gql`
+        extend type Lot {
+          lot: AuctionsLotState!
+        }
+      `,
+      resolvers: {
+        Lot: {
+          lot: {
+            resolve: (parent, args, context) => {
+              console.log("**********lot:", context._lot)
+              return context._lot
```

Comment:
As of yesterday devening this is where @zephraph, @xtina-starr and myself have left off:
- We cannot specify the new `Lot` type to have a `AuctionsLotState` field in the local schema because that type comes from causality
- we can do it here - we are assuming that the supplised object will be a `{saleArtwork: SaleArtwork, lot: AuctionsLotState}`, but the `parent` arg above only contains the fields selected in that query at this step (even though we are supplying a full sale artwork and lot in our hardcoded query result below, only the fields we explicitly query are visible here).
- As a result in our actual resolver we are setting a secret property called `_lot` as a hack to confirm our suspicions.

I can't remember if there is any additional context or things we've tried or planned to try yet.